specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc

```diff
@@ -885,10 +882,9 @@ descriptor to specify a unique name for one of the modules.
 of all components in every module contained in the application. For each
 component dependency, there must only be one corresponding component
 that fulfills that dependency in the scope of the application.
-. For each _ejb-link_ , there must be only one
-matching _ejb-name_ in the scope of the entire application (see
-link:#a936[See Enterprise JavaBeans™
-(EJB) References]”).
+. For each _enterprisebean-link_ , there must be only one
```

Comment:
These links being referenced (here and earlier) are referring to the dependencies...  I don't think these need to change...

Reply:
reverted